test/fixtures/hello-cloud/stateful_set.yml

```diff
@@ -11,9 +11,9 @@ spec:
         app: hello-cloud
         name: nginx-ss
     spec:
+      terminationGracePeriodSeconds: 0
       containers:
       - name: nginx
-        image: gcr.io/google_containers/nginx-slim:0.8
-        command: ["sleep", "40"]
+        image: nginx:alpine
```

Comment:
Many other tests use this image, so why pull a different one from GCR just for this test?

Reply:
Do we actually care that it be Nginx? If not, we could use an even smaller image (e.g. `busybox`) but this is probably good enough.

Reply:
No, we don't really care. OTOH it should only be pulled once per run, so the pull time's impact should be minimal. Is there a particular busybox command you think would work nicely? I'm going to merge this now since it is a big improvement, but I can follow up to switch images on everything.

Reply:
If you just want it to stay up forever (like Nginx would) then `tail -f /dev/null` is a good trick. IIRC the Busybox image is 2MB.